fr/blog.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: blog
+lang: fr
+title: Blog-fr
+name: blog-fr
+permalink: /fr/blog/
+share: false
+version: 1
+---
```

Comment:
For consistency to other translations, can you add a French translation text of this placeholder text for this page?

> Would you like to help translate our publications? See the [CONTRIBUTING documentation](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/CONTRIBUTING.md#translations) and the [French translation issues and PRs](https://github.com/bitcoinops/bitcoinops.github.io/pulls?&q=label%3Alocalization-french) in our github repo.

Reply:
OK, done

Reply:
Pushed a commit to put in the links on this page

Reply:
Hello,

Does all is OK now ?